slingr-stack/ftp-service
# 15545 Migrate FTP endpoint to service

Pull-request for issue https://github.com/slingr-stack/platform/issues/15545 created by @pasaperez-slingr 

## What it does?

Migrate the FTP endpoint to Services.
This migration has been done with Java 17

## How to test it?

You can clone this repository and register the ftp-service with the test repository `https://github.com/pasaperez-slingr/ftp-service` already has a tag defined. If testing locally, set the credentials that test4 has and make sure to use java version 17 as SDK module.

Remember to use this [checklist](https://docs.google.com/document/d/139rKhIsvpT3yBSB88BBZfy03fD_Sried0bVwIWVaHJk/edit?usp=sharing) for the review.

## Technical notes

There aren't technical notes

## Deployment notes

Create a new tag of ftp-service and register on each environment. Make a build of this tag.
